frontend/packages/topology/src/filters/TopologyFilterBar.tsx

```diff
@@ -198,12 +199,25 @@ const TopologyFilterBar: React.FC<TopologyFilterBarProps> = ({
           variant={ToolbarGroupVariant['button-group']}
           alignment={{ default: 'alignRight' }}
         >
+          <ToolbarItem
+            className={
+              isExportAppEnabled || kialiLink
+                ? 'odc-topology-filter-bar__resource-quota-warning-block'
+                : ''
+            }
+          >
+            <ResourceQuotaAlert namespace={namespace} />
+          </ToolbarItem>
           {kialiLink && (
             <ToolbarItem className="odc-topology-filter-bar__kiali-link1">
               <ExternalLink href={kialiLink} text={t('topology~Kiali')} />
             </ToolbarItem>
           )}
-          <ExportApplication namespace={namespace} isDisabled={isDisabled} />
+          <ExportApplication
+            namespace={namespace}
+            isDisabled={isDisabled}
+            setIsExportAppEnabled={setIsExportAppEnabled}
+          />
```

Comment:
can we avoid this and directly make use of hook `useFlag(ALLOW_EXPORT_APP)`

Reply:
updated